### ISSUE TITLE
Latest version breaks my build

### ISSUE THREAD

**Issue by OriR** [storybooks/storybook-addon-console#8]
I'm getting this error:
```
info @storybook/react v3.3.15
info 
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (extending mode).
info Building storybook ...
ERR! Failed to build the storybook
ERR! Module not found: Error: Can't resolve '@babel/runtime/helpers/toConsumableArray' in '<package>/node_modules/@storybook/addon-console/dist'
```

Sometimes it's
```
info @storybook/react v3.3.15
info
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (extending mode).
info Building storybook ...
ERR! Failed to build the storybook
ERR! Module not found: Error: Can't resolve '@babel/runtime/helpers/inherits' in '<package>/node_modules/@storybook/addon-console/dist'
```

This started happening the day before yesterday, I also saw that the latest version is from 4 days ago and it updated `babel`.

These are my storybook dependencies:
```
    "@dump247/storybook-state": "^1.5.0",
    "@storybook/addon-actions": "3.3.15",
    "@storybook/addon-console": "^1.0.4",
    "@storybook/addon-knobs": "3.3.15",
    "@storybook/addon-links": "3.3.15",
    "@storybook/addon-options": "3.3.15",
    "@storybook/addon-storysource": "3.4.8",
    "@storybook/addon-viewport": "3.4.8",
    "@storybook/cli": "3.3.15",
    "@storybook/react": "3.3.15",
    "storybook-addon-versions": "^0.4.3",
    "storybook-readme": "3.3.0",
```

**Comment by UsulPro** [comment]
Hi @OriR Thanks for reporting!
Could you clarify what version of Storybook do you use?

**Comment by UsulPro** [comment]
Sorry for the inconvenience, it's fixed in v 1.0.10 now. 
Let me know, please, if there are still left issues.
